ouroboros-consensus-diffusion/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 # Changelog entries
 
+<a id='changelog-0.8.0.1'></a>
+## 0.8.0.1 — 2023-09-27
+
+### Non-Breaking
```

Comment:
Also, `Patch`, right? Just based on the version number you chose.

Or maybe the version bump is too gentle.

Reply:
https://github.com/input-output-hk/ouroboros-consensus/pull/377#discussion_r1339685322